### PR TITLE
Change the BG color of the dashboards list page

### PR DESCRIPTION
**Pull request by mazameli**
Trying to figure out how to make the filter box transparent still.

### CODE REVIEW

File: frontend/src/metabase/components/SearchHeader.jsx [hunk 1]
```diff
@@ -11,7 +11,7 @@ const SearchHeader = ({ searchText, setSearchText }) =>
     <div className={S.searchHeader}>
         <Icon className={S.searchIcon} name="search" size={18} />
         <input
-            className={cx("input", S.searchBox)}
+            className={cx("input bg-transparent", S.searchBox)}
```
Comment (attekei) [inline frontend/src/metabase/components/SearchHeader.jsx]: Good to use a reusable css class instead of just adding new styles to the `.searchBox` class!

**Review by attekei — COMMENTED** [review]
Looks good! Is it intentional that the page for a single dashboard has a white header background but the dashboard list does not?

We should probably make the loading wrapper transparent as well:
screen shot 2017-05-05 at 16 04 52

File: frontend/src/metabase/components/SearchHeader.jsx (hunk 1 above)
Comment (mazameli) [inline frontend/src/metabase/components/SearchHeader.jsx]: That's all thanks to @kdoh :)

**Comment by mazameli** [comment]
@attekei @kdoh I _think_ I made the loading background transparent correctly, and I checked around the rest of the app to see if it busted any other loading spinners and it seems cool, but would love your eyes on this real quick.

**Comment by attekei** [comment]
I noticed that `<LoadingAndErrorWrapper>` actually has a `noBackground` property that does what we want. It was so simple modification that I committed it right away. In my opinion this is ready to be merged after the tests have passed (they are still running).

**Comment by mazameli** [comment]
Resolves #4999

**Pull request merged**